.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,122 @@
+        Provide 'git' version or 'other' in description
+      options:
+        - label: I have checked releases and am using the latest release.
+          required: true
+  - type: input
+    id: verinfo
+    attributes:
+      label: Different version than latest?
+      description: Alpha, Git, or Branch?
+      placeholder: "Ex. 0.78.0-alpha, git verion, branch testing."
+    validations:
+      required: false
+  - type: dropdown
+    id: OS
+    attributes:
+      label: What Operating System are you using?
+      multiple: false
+      options:
+        - Windows 11
+        - Windows 10
+        - Windows 8
+        - Windows 7
+        - Windows Other
+        - Mac OS
+        - Linux x86
+        - Linux x86_64
+        - Linux ARM (Raspberry Pi)
+        - Other
+    validations:
+      required: false
+  - type: input
+    id: other
+    attributes:
+      label: If Other OS, please describe
+      description: Other details
+      placeholder: "Windows, Mac OSX version, Debian, Ubuntu, Arch, etc."
+    validations:
+      required: false
+  - type: input
+    id: hardware
+    attributes:
+      label: Relevant hardware info
+      description: Hardware
+      placeholder: "CPU, GPU, device brand/model: e.g. Raspberry Pi3B+, Nvidia, etc"
+    validations:
+      required: false
+  - type: checkboxes
+    id: checked
+    attributes:
+      label: Have you checked that no other similar issue already exists?
+      description: Searched issues before creating report?
+      options:
+        - label: I have searched and not found similar issues.
+          required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: A clear and concise description of what the bug is.
+      description: Describe what happens, what software were you running? _Include screenshot if possible_
+      placeholder: "How & When does this occur?"
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to reproduce the behaviour.
+      description: How can we reproduce this?
+      value: |
+        Explain how to reproduce
+        1. 
+        2.
+        3.
+    validations:
+      required: false
+  - type: textarea
+    id: config_file
+    attributes:
+      label: Your configuration
+      description: |
+        Share the config file(s) you've been using to run the program. (`dosbox-staging.conf`) 
+        Please avoid pasting the full config, _use attachments or links_ in a [Gist](https://gist.github.com/)
+      placeholder: "example: texture_renderer = opengl"
+      render: ini # syntax highlighting
+    validations:
+      required: false
+  - type: textarea
+    id: log
+    attributes:
+      label: Provide a Log
+      description: Please avoid pasting the full log, _use attachments or links_ in a [Gist](https://gist.github.com/)
```

see next line. if you did that change you should probably edit the description to something like:
```suggestion
      description: Please paste the full log produced by the emulator here
```

---

i was definately trying to highlight an easy way for someone to post a large log, suggesting gist as the quickest

was there a difference somewhere?

---

Actually i'll just drop a link: https://github.com/dosbox-staging/dosbox-staging/pull/1235#discussion_r701978966
If you don't do that change you can ignore this